### ISSUE TITLE
adferrand/backuppc:4.3.0-4 fails to start - You must type in 4 to 1023 characters 

### ISSUE THREAD

**Issue by anders-larsson** [adferrand/docker-backuppc#20]
Hi,

Just attempted to upgrade to use 4.3.0-4 however it fails to start with the error below. I'm not sure why it's re-generating RSA private key though since it's being upgraded? How should I proceed from here?

```
2019-03-10 08:30:54 | stdout | Enjoy!
2019-03-10 08:30:55 | stdout | Generating RSA private key, 2048 bit long modulus (2 primes)
2019-03-10 08:30:56 | stdout | .........................................................................................................+++++
2019-03-10 08:30:56 | stdout | .........................+++++
2019-03-10 08:30:56 | stdout | 140689475521384:error:0906906F:PEM routines:PEM_ASN1_write_bio:read key:crypto/pem/pem_lib.c:357:
-- | -- | --
2019-03-10 08:30:56 | stdout | 140689475521384:error:28078065:UI routines:UI_set_result_ex:result too small:crypto/ui/ui_lib.c:903:You must type in 4 to 1023 characters
2019-03-10 08:30:56 | stdout | 140689475521384:error:28078065:UI routines:UI_set_result_ex:result too small:crypto/ui/ui_lib.c:903:You must type in 4 to 1023 characters
```
Thanks

**Comment by anders-larsson** [comment]
It's probably re-generating the certificate because I have environment variable `USE_SSL=true` but /etc/lighthttpd is not mapped to a volume. It still should not fail to generate the certificate?

**Comment by adferrand** [comment]
Indeed, it should not fail. I fixed that with 4.3.0-5. Thanks a lot for your report!

**Comment by anders-larsson** [comment]
It works much better with -5.

Thank you for providing BackupPC in this way.
